src/vcpkg/metrics.cpp

```diff
@@ -133,6 +133,8 @@ namespace vcpkg
         {StringMetric::CommandName, "command_name", "z-preregister-telemetry"},
         {StringMetric::DeploymentKind, "deployment_kind", "Git"},
         {StringMetric::DetectedCiEnvironment, "detected_ci_environment", "Generic"},
+        {StringMetric::GithubRepositoryId, "github_repository_id", "0"},
+        {StringMetric::GithubRepositoryOwnerId, "github_repository_owner_id", "0"},
```

Comment:
```suggestion
        {StringMetric::CiProjectId, "ci_project_id", "0"},
        {StringMetric::CiOwnerId, "ci_owner_id", "0"},
```

I'd prefer to have generic names in case we extend this to other CI vendors.